libs/blockchain-watcher/feature-app/src/lib/Klass/PaymentCreatedEvent.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { PaymentCreatedEvent } from './PaymentCreatedEvent'
+import { v4 } from 'uuid'
+import _ from 'lodash'
+import { EnumPaymentCreatedEventType } from '../Enum/EnumPaymentCreatedEventType'
+import { stringHelper } from '@solomon/blockchain-watcher/feature-app'
+
+describe('PaymentCreatedEvent', () => {
+  it('constructor() with empty props', () => {
+    let p = new PaymentCreatedEvent({})
+
+    expect(p).toBeInstanceOf(PaymentCreatedEvent)
+  })
+
+  it('constructor() with valid props', () => {
+    let id = v4()
```

Comment:
Any reason for `let` instead of `const`? This is a general comment for all usage. I'm not sure why the linter doesn't catch this.

Reply:
My bad habit :(

Reply:
Not a problem, though I will make a note to check the linter so we don't have to think about it.